app/views/CreditAndRefunds.scala.html

```diff
@@ -77,36 +70,107 @@
-            <span class="govuk-body">@messages("credit-and-refund.subHeading.has-credits-1") </span>
-            @maybeAvailableCredit.toCurrency
-            <!--Checks if there are any items in the list, depending on that ":" or "." message is used.-->
-            @{
-                val listSize = viewModel.creditCharges.filter(x => {
-                        if(x._2.getCreditType == Some(MfaCreditType))
-                            isMFACreditsAndDebitsEnabled
-                        else if(x._2.getCreditType == Some(CutOverCreditType))
-                            isCutOverCreditsEnabled
-                        else
-                            true
-                    }).size
-                if( listSize != 1 || balance.get.firstPendingAmountRequested != None || balance.get.secondPendingAmountRequested != None){
-                    <span class="govuk-body">{messages(if(isAgent) "credit-and-refund.agent.subHeading.has-credits-2" else "credit-and-refund.subHeading.has-credits-2")}</span>
-                }else{
-                    <span class="govuk-body">{messages(if(isAgent) "credit-and-refund.agent.subHeading.has-credits-3" else "credit-and-refund.subHeading.has-credits-3")}</span>
+            <span class="govuk-body">@messages("credit-and-refund.subHeading.has-no-credits-1")</span>
+                @{
+                balance.get.firstPendingAmountRequested.getOrElse(throw MissingFieldException("First Pending Amount")).toCurrency
                 }
-            }
+            <span class="govuk-body"> @messages("credit-and-refund.subHeading.has-no-credits-2")</span>
         </h2>
-    } else {
-            <h2 class="govuk-heading-s">
-                <span class="govuk-body">@messages("credit-and-refund.subHeading.has-no-credits-1")</span>
-                    @{
-                    balance.get.firstPendingAmountRequested.getOrElse(throw MissingFieldException("First Pending Amount")).toCurrency
-                    }
-                <span class="govuk-body"> @messages("credit-and-refund.subHeading.has-no-credits-2")</span>
-            </h2>
+    }
+}
+
+@getCreditAndRefundListContent = {
+    <details class="govuk-details">
+        <summary class="govuk-details__summary">
+            <span class="govuk-details__summary-text">
+              @messages("credit-and-refund.subHeading-source")
+            </span>
+        </summary>
+        <div class="govuk-details__text">
+            @creditAndRefundType match {
+                case Some(UnallocatedCreditFromOnePayment) => {
+                    <h2 class="govuk-heading-s">
+                        <span class="govuk-body" id="unallocated-payment">
+                            <strong class="govuk-body govuk-!-font-weight-bold">@maybeAvailableCredit.toCurrencyString</strong>
+                            @messages("credit-and-refund.payment")
+                            @maybeCreditChargesHead._1.dueDate.getOrElse(throw MissingFieldException("Due Date")).toLongDate
```

Comment:
if we can avoid using tuples in this way, (._1, _2), as this make code not readable